packages/jest-diff/src/__tests__/diff.test.ts

```diff
@@ -192,24 +192,26 @@ line 4`;
 });
 
 describe('objects', () => {
-  const a = {a: {b: {c: 5}}};
-  const b = {a: {b: {c: 6}}};
-  const expected = [
-    '  Object {',
-    '    "a": Object {',
-    '      "b": Object {',
-    '-       "c": 5,',
-    '+       "c": 6,',
-    '      },',
-    '    },',
-    '  }',
-  ].join('\n');
+  describe('expand', () => {
+    const a = {a: {b: {c: 5}}};
+    const b = {a: {b: {c: 6}}};
+    const expected = [
+      '  Object {',
+      '    "a": Object {',
+      '      "b": Object {',
+      '-       "c": 5,',
+      '+       "c": 6,',
+      '      },',
+      '    },',
+      '  }',
+    ].join('\n');
 
-  test('(unexpanded)', () => {
-    expect(diff(a, b, unexpandedBe)).toBe(expected);
-  });
-  test('(expanded)', () => {
-    expect(diff(a, b, expandedBe)).toBe(expected);
+    test('(unexpanded)', () => {
+      expect(diff(a, b, unexpandedBe)).toBe(expected);
+    });
+    test('(expanded)', () => {
+      expect(diff(a, b, expandedBe)).toBe(expected);
+    });
   });
 });
```

Comment:
@WeiAnAn If it is not too much trouble to undo the change to this test in `jest-diff` package, I think it will be clearer that the final design was to encapsulate the improvement within `jest-matcher-utils` package.